get_empirical_dist.R

```diff
@@ -50,6 +50,13 @@ option_list = list(
         type = 'integer',
         help = 'Number of sampling iterations to construct empirical distribution'
     ),
+    make_option(
+        c("-c", "--sample-prop"),
+        action = "store",
+        default = 0.1,
+        type = 'float',
```

Comment:
@rubenchazarra,  I'm not sure you can use `float` here - see the error logs in Travis

Reply:
Yeah my bad, it's 'numeric' that should be there